src/ext/configuration.c

```diff
@@ -131,3 +134,80 @@ DD_CONFIGURATION
 #undef BOOL
 #undef INT
 #undef DOUBLE
+
+bool ddtrace_config_bool(ddtrace_string subject, bool default_value) {
+    ddtrace_string str_1 = {
+        .ptr = "1",
+        .len = 1,
+    };
+    ddtrace_string str_true = {
+        .ptr = "true",
+        .len = sizeof("true") - 1,
+    };
+    if (ddtrace_string_equals(subject, str_1) || ddtrace_string_equals(subject, str_true)) {
+        return true;
+    }
+    ddtrace_string str_0 = {
+        .ptr = "0",
+        .len = 1,
+    };
+    ddtrace_string str_false = {
+        .ptr = "false",
+        .len = sizeof("false") - 1,
+    };
+    if (ddtrace_string_equals(subject, str_0) || ddtrace_string_equals(subject, str_false)) {
+        return false;
+    }
+    return default_value;
+}
+
+bool ddtrace_config_distributed_tracing_enabled(TSRMLS_D) {
+    ddtrace_string distributed_tracing_enabled =
+        ddtrace_string_cstring_ctor(ddtrace_getenv(ZEND_STRL("DD_DISTRIBUTED_TRACING") TSRMLS_CC));
```

Comment:
Can `ddtrace_string_getenv` be used here ?

Reply:
Probably -- I added `ddtrace_string_getenv` after I noticed the repetition and probably just missed this one.